x-pack/plugins/security_solution/public/common/components/event_details/cti_details/host_risk_summary.tsx

```diff
@@ -54,7 +54,7 @@ const HostRiskSummaryComponent: React.FC<{
       {hostRisk.isModuleEnabled && hostRisk.result && hostRisk.result.length > 0 && (
         <>
           <EnrichedDataRow
-            field={'host.risk.keyword'}
+            field={i18n.HOST_RISK_CLASSIFICATION}
```

Comment:
NIT: the `field` prop name is a little bit misleading, maybe we should call it `title` or `label`